### PR TITLE
add 6.1.3 release notes

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/11445
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.3.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
++ TiFlash
+
+    <!--compute owner: zanmato1984 -->
+
+    - (dup) 修复逻辑运算符在 UInt8 类型下查询结果出错的问题 [#6127](https://github.com/pingcap/tiflash/issues/6127)
+    - (dup) 修复 `CAST(value AS datetime)` 输入数据无法转成 `DATETIME` 时会导致 TiFlash sys CPU 异常高的问题 [#5097](https://github.com/pingcap/tiflash/issues/5097) @[xzhangxian1008](https://github.com/xzhangxian1008)
+
+    <!--storage owner: flowbehappy -->
+
+    - Fix too many column files under heavy write pressure. [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
```
Comment (flowbehappy) [inline releases/release-6.1.3.md:58]:
```suggestion
    - 修复高压力写可能产生太多小文件的问题 [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
```

File: releases/release-6.1.3.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    - Fix the problem that column files in delta layer cannot be compacted after restart tiflash [#6159](https://github.com/pingcap/tiflash/issues/6159) @[lidezhu](https://github.com/lidezhu)
```
Comment (flowbehappy) [inline releases/release-6.1.3.md:59]:
```suggestion
    - 修复 TiFlash 重启后 delta 层小文件无法合并的问题 [#6159](https://github.com/pingcap/tiflash/issues/6159) @[lidezhu](https://github.com/lidezhu)
```

**Review by flowbehappy — APPROVED** [review]
LGTM

File: releases/release-6.1.3.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+# TiDB 6.1.3 Release Notes
+
+发版日期：2022 年 xx 月 xx 日
+
+TiDB 版本：6.1.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.1.3#version-list)
+
+## 提升改进
+
```
Comment (nolouch) [inline releases/release-6.1.3.md:14]:
```suggestion
- PD
   - 优化锁粒度，提升高并发下心跳的处理能力 [#5635](https://github.com/tikv/pd/pull/5635) @[rleungx](https://github.com/rleungx)
```

File: releases/release-6.1.3.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    <!--owner: @niubell-->
+
+    + TiDB Data Migration (DM)
+
+    <!--owner: @niubell-->
+
+        - Fix the issue that when `collation_compatible` is set to `"strict"`, DM might generate SQL with duplicated collations [#6832](https://github.com/pingcap/tiflow/issues/6832) @[lance6716](https://github.com/lance6716)
+        - Fix sometime DM task is stopped with error "Unknown placement policy" [#7493](https://github.com/pingcap/tiflow/issues/7493) @[lance6716](https://github.com/lance6716)
+
```
Comment (lance6716) [inline releases/release-6.1.3.md:89]:
```suggestion
        - Fix in some cases log contains many "COMMIT" related lines and relay log file may re-pull from upstream [#7719](https://github.com/pingcap/tiflow/pull/7719) @[liumengya94](https://github.com/liumengya94)
        - Fix when DM worker exits a new worker may be scheduled too quickly and write duplicated data [#7745](https://github.com/pingcap/tiflow/pull/7745) @[GMHDBJD](https://github.com/GMHDBJD)

```

File: releases/release-6.1.3.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+# TiDB 6.1.3 Release Notes
+
+发版日期：2022 年 xx 月 xx 日
+
+TiDB 版本：6.1.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.1.3#version-list)
+
+## 提升改进
+
```
Comment (nongfushanquan) [inline releases/release-6.1.3.md:14]: - Enable transaction split and disable safeMode of a changefeed in TiCDC by defult. [#7505](https://github.com/pingcap/tiflow/issues/7505)
 - Improve kakfa protocol encoder performance.[#7540](https://github.com/pingcap/tiflow/issues/7540)[#7532](https://github.com/pingcap/tiflow/issues/7532)[#7543](https://github.com/pingcap/tiflow/issues/7532)

File: releases/release-6.1.3.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    - (dup) 修复错误下推的条件被 Join Reorder 丢弃后导致查询结果错误的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - Fix an issue that `get_lock()` cannot hold for more than 10 minutes. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
+    - Fix the issue that the auto increment column cannot be used with check constraint. [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
+    - Fix broken log rotation of grpc [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
+    - Delete TiFlash sync status from etcd when table is truncated or dropped [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
+    - Fix the issue of arbitrary file read via data source name injection (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
+
+    <!--executor owner: zanmato1984-->
+
+    <!--planner owner: qw4990-->
```
Comment (qw4990) [inline releases/release-6.1.3.md:31]:
```suggestion
    - 修复 Projection 算子错误下推表达式到 Join 导致结果错误的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[[winoros](https://github.com/winoros)](https://github.com/winoros)
    - 修复统计信息后台任务搜集分区表统计信息时可能崩溃的问题 [#35421](https://github.com/pingcap/tidb/issues/35421) @[[lilinghai](https://github.com/lilinghai)](https://github.com/lilinghai)
```

File: releases/release-6.1.3.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    <!--owner: @niubell-->
+
+    + TiDB Data Migration (DM)
+
+    <!--owner: @niubell-->
+
+        - Fix the issue that when `collation_compatible` is set to `"strict"`, DM might generate SQL with duplicated collations [#6832](https://github.com/pingcap/tiflow/issues/6832) @[lance6716](https://github.com/lance6716)
+        - Fix sometime DM task is stopped with error "Unknown placement policy" [#7493](https://github.com/pingcap/tiflow/issues/7493) @[lance6716](https://github.com/lance6716)
+        - Fix in some cases log contains many "COMMIT" related lines and relay log file may re-pull from upstream [#7719](https://github.com/pingcap/tiflow/pull/7719) @[liumengya94](https://github.com/liumengya94)
+        - Fix when DM worker exits a new worker may be scheduled too quickly and write duplicated data [#7745](https://github.com/pingcap/tiflow/pull/7745) @[GMHDBJD](https://github.com/GMHDBJD)
```
Comment (lance6716) [inline releases/release-6.1.3.md:90]:
```suggestion
        - 修复 `collation_compatible` 设置为 `"strict"` 时 DM 生成 SQL 时可能带有重复的 COLLATE 的问题 [#6832](https://github.com/pingcap/tiflow/issues/6832) @[lance6716](https://github.com/lance6716)
        - 修复 DM 可能由于 "Unknown placement policy" 错误导致任务暂停的问题 [#7493](https://github.com/pingcap/tiflow/issues/7493) @[lance6716](https://github.com/lance6716)
        - 修复在某些场景下日志中出现大量包含 "COMMIT" 行，并且 relay log 文件会从上游重新拉取的问题 [#7719](https://github.com/pingcap/tiflow/pull/7719) @[liumengya94](https://github.com/liumengya94)
        - 修复当 DM worker 即将退出时新 worker 调度过快导致数据被重复同步的问题 [#7745](https://github.com/pingcap/tiflow/pull/7745) @[GMHDBJD](https://github.com/GMHDBJD)
```

File: releases/release-6.1.3.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+    - 修复高压力写可能产生太多 delta 层小文件的问题 [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
+    - 修复 TiFlash 重启后 delta 层小文件无法合并的问题 [#6159](https://github.com/pingcap/tiflash/issues/6159) @[lidezhu](https://github.com/lidezhu)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+    <!--owner: @3pointer-->
+
+        - Fix a bug that: "Error Unsupported collation when new collation is enabled: 'utf8mb4_0900_ai_ci'" even if new_collation_enabled is false. [#39150](https://github.com/pingcap/tidb/issues/39150) @[MoCuishle28](https://github.com/MoCuishle28)
```
Comment (WangLe1321) [inline releases/release-6.1.3.md:77]:
```suggestion
        - 修复数据库中使用旧的 collation 时恢复失败的问题. [#39150](https://github.com/pingcap/tidb/issues/39150) @[MoCuishle28](https://github.com/MoCuishle28)
```

File: releases/release-6.1.3.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+    - Fix an issue that `get_lock()` cannot hold for more than 10 minutes. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
+    - Fix the issue that the auto increment column cannot be used with check constraint. [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
+    - Fix broken log rotation of grpc [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
+    - Delete TiFlash sync status from etcd when table is truncated or dropped [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
+    - Fix the issue of arbitrary file read via data source name injection (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
+
+    <!--executor owner: zanmato1984-->
+
+    - 修复 Projection 算子错误下推表达式过 Join 算子的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - 修复统计信息后台任务搜集分区表统计信息时可能崩溃的问题 [#35421](https://github.com/pingcap/tidb/issues/35421) @[lilinghai](https://github.com/lilinghai)
```
Comment (zanmato1984) [inline releases/release-6.1.3.md:41]:
```suggestion
    - 修复统计信息后台任务搜集分区表统计信息时可能崩溃的问题 [#35421](https://github.com/pingcap/tidb/issues/35421) @[lilinghai](https://github.com/lilinghai)
    - 修复函数 `str_to_date` 在 `NO_ZERO_DATE` SQL Mode 下返回结果不正确的问题 [#39146](https://github.com/pingcap/tidb/issues/39146) @[mengxin9014](https://github.com/mengxin9014)
```

File: releases/release-6.1.3.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+    <!--owner: @niubell-->
+
+    + TiDB Data Migration (DM)
+
+    <!--owner: @niubell-->
+
+        - 修复 `collation_compatible` 设置为 `"strict"` 时 DM 生成 SQL 时可能带有重复的 COLLATE 的问题 [#6832](https://github.com/pingcap/tiflow/issues/6832) @[lance6716](https://github.com/lance6716)
+        - 修复 DM 可能由于 "Unknown placement policy" 错误导致任务暂停的问题 [#7493](https://github.com/pingcap/tiflow/issues/7493) @[lance6716](https://github.com/lance6716)
+        - 修复在某些场景下日志中出现大量包含 "COMMIT" 行，并且 relay log 文件会从上游重新拉取的问题 [#7719](https://github.com/pingcap/tiflow/pull/7719) @[liumengya94](https://github.com/liumengya94)
```
Comment (lance6716) [inline releases/release-6.1.3.md:99]:
```suggestion
        - 修复在某些场景下 relay log 文件会从上游重新拉取的问题 [#7719](https://github.com/pingcap/tiflow/pull/7719) @[liumengya94](https://github.com/liumengya94)
```

**Review by tonyxuqqi — APPROVED** [review]
LGTM on tikv part

File: releases/release-6.1.3.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+    - (dup) 修复 `mysql.tables_priv` 表中 `grantor` 字段缺失的问题 [#38293](https://github.com/pingcap/tidb/issues/38293) @[CbcWestwolf](https://github.com/CbcWestwolf)
+    - (dup) 修复错误下推的条件被 Join Reorder 丢弃后导致查询结果错误的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - Fix an issue that `get_lock()` cannot hold for more than 10 minutes. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
+    - Fix the issue that the auto increment column cannot be used with check constraint. [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
+    - Fix broken log rotation of grpc [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
+    - Delete TiFlash sync status from etcd when table is truncated or dropped [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
+    - Fix the issue of arbitrary file read via data source name injection (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
```
Comment (hawkingrei) [inline releases/release-6.1.3.md:36]:
```suggestion
    - 修复了 `get_lock()` 不能持续 10 分钟的问题. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
    - 修复了自增列不能和检查约束一起使用的问题 [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
    - 修复了损坏的 grpc log rotation [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
    - 当表被截断或删除时从 etcd 中删除 TiFlash 同步状态 [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
    - 修复通过数据源名称注入读取任意文件的问题 (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
```

File: releases/release-6.1.3.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+    <!--sql-infra and tidb owner: hawkingrei-->
+
+    - (dup) 修复 `mysql.tables_priv` 表中 `grantor` 字段缺失的问题 [#38293](https://github.com/pingcap/tidb/issues/38293) @[CbcWestwolf](https://github.com/CbcWestwolf)
+    - (dup) 修复错误下推的条件被 Join Reorder 丢弃后导致查询结果错误的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - 修复了 `get_lock()` 不能持续 10 分钟的问题. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
+    - 修复了自增列不能和检查约束一起使用的问题 [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
+    - 修复了损坏的 grpc log rotation [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
+    - 当表被截断或删除时从 etcd 中删除 TiFlash 同步状态 [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
+    - 修复通过数据源名称注入读取任意文件的问题 (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
```
Comment (shichun-0415) [inline releases/release-6.1.3.md:42]:
```suggestion
    - 修复 `get_lock()` 无法持续 10 分钟以上的问题 [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
    - 修复自增列不能和检查约束一起使用的问题 [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
    - 修复了 gRPC 日志导出到错误文件的问题 [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
    - 修复当表被截断或删除时 TiFlash 同步状态未从 etcd 中删除的问题 [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
    - 修复通过数据源名称注入可读取任意文件的问题 (CVE-2022-3023) [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
```

File: releases/release-6.1.3.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+
+    <!--executor owner: zanmato1984-->
+
+    - 修复函数 `str_to_date` 在 `NO_ZERO_DATE` SQL Mode 下返回结果不正确的问题 [#39146](https://github.com/pingcap/tidb/issues/39146) @[mengxin9014](https://github.com/mengxin9014)
+
+    <!--planner owner: qw4990-->
+
+    - 修复统计信息后台任务搜集分区表统计信息时可能崩溃的问题 [#35421](https://github.com/pingcap/tidb/issues/35421) @[lilinghai](https://github.com/lilinghai)
```
Comment (shichun-0415) [inline releases/release-6.1.3.md:50]:
```suggestion
    - 修复后台统计信息任务可能崩溃的问题 [#35421](https://github.com/pingcap/tidb/issues/35421) @[lilinghai](https://github.com/lilinghai)
```

File: releases/release-6.1.3.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.1.3#version-list)
+
+## 提升改进
+
+- PD
+
+<!--owner: nolouch -->
+
+    - 优化锁粒度，提升高并发下心跳的处理能力 [#5586](https://github.com/tikv/pd/issues/5586) @[rleungx](https://github.com/rleungx)
```
Comment (TomShawn) [inline releases/release-6.1.3.md:19]:
```suggestion
    - 优化锁的粒度以减少锁争用，提升高并发下心跳的处理能力 [#5586](https://github.com/tikv/pd/issues/5586) @[rleungx](https://github.com/rleungx)
```

File: releases/release-6.1.3.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+
+    + Dumpling
+
+    <!--owner: @niubell-->
+
+    + TiCDC
+
+        <!--owner: @nongfushanquan-->
+
+        - 修复在执行 DDL 时，恢复 changefeed 会导致数据丢失的问题 [#7682](https://github.com/pingcap/tiflow/issues/7682) @[asddongmen](https://github.com/asddongmen)
```
Comment (shichun-0415) [inline releases/release-6.1.3.md:96]:
```suggestion
        - 修复在执行 DDL 后，暂停然后恢复 changefeed 会导致数据丢失的问题 [#7682](https://github.com/pingcap/tiflow/issues/7682) @[asddongmen](https://github.com/asddongmen)
```

File: releases/release-6.1.3.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+
+    - 优化锁粒度，提升高并发下心跳的处理能力 [#5586](https://github.com/tikv/pd/issues/5586) @[rleungx](https://github.com/rleungx)
+
+- Tools
+
+    - TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+        - 修改同步任务 safe-mode 和拆分大事务默认值提升性能  [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
```
Comment (TomShawn) [inline releases/release-6.1.3.md:27]:
```suggestion
        - 开启大事务拆分，默认关闭同步任务的安全模式以提升性能  [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
```

File: releases/release-6.1.3.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+        - 提升 Kafka 相关协议的编码性能. [#7540](https://github.com/pingcap/tiflow/issues/7540), [#7532](https://github.com/pingcap/tiflow/issues/7532), [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
```
Comment (TomShawn) [inline releases/release-6.1.3.md:28]:
```suggestion
        - 提升 Kafka 相关协议的编码性能 [#7540](https://github.com/pingcap/tiflow/issues/7540), [#7532](https://github.com/pingcap/tiflow/issues/7532), [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
```

File: releases/release-6.1.3.md [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    <!--sql-infra and tidb owner: hawkingrei-->
+
+    - (dup) 修复 `mysql.tables_priv` 表中 `grantor` 字段缺失的问题 [#38293](https://github.com/pingcap/tidb/issues/38293) @[CbcWestwolf](https://github.com/CbcWestwolf)
+    - (dup) 修复错误下推的条件被 Join Reorder 丢弃后导致查询结果错误的问题 [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - 修复 `get_lock()` 无法持续 10 分钟以上的问题 [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
```
Comment (TomShawn) [inline releases/release-6.1.3.md:38]:
```suggestion
    - 修复使用 `get_lock()` 获得的锁无法持续 10 分钟以上的问题 [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
```

File: releases/release-6.1.3.md [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+    - 优化锁的粒度以减少锁争用，提升高并发下心跳的处理能力 [#5586](https://github.com/tikv/pd/issues/5586) @[rleungx](https://github.com/rleungx)
+
+- Tools
+
+    - TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+        - 开启大事务拆分，默认关闭同步任务的安全模式以提升性能 [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
+        - 提升 Kafka 相关协议的编码性能 [#7540](https://github.com/pingcap/tiflow/issues/7540), [#7532](https://github.com/pingcap/tiflow/issues/7532), [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
```
Comment (Oreoxmt) [inline releases/release-6.1.3.md:28]:
```suggestion
        - 提升 Kafka 相关协议的编码性能 [#7540](https://github.com/pingcap/tiflow/issues/7540) [#7532](https://github.com/pingcap/tiflow/issues/7532) [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
```

File: releases/release-6.1.3.md [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+
+    <!--owner: nolouch -->
+
+    - (dup) 修复 Stream 超时问题，提高 Leader 切换的速度 [#5207](https://github.com/tikv/pd/issues/5207) @[CabinfeverB](https://github.com/CabinfeverB)
+
++ TiKV
+
+    <!--owner: tonyxuqqi-->
+
+    - (dup) 修复获取 Snapshot 时 Lease 过期引发的异常竞争问题  [#13553](https://github.com/tikv/tikv/issues/13553) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
```
Comment (Oreoxmt) [inline releases/release-6.1.3.md:70]:
```suggestion
    - (dup) 修复获取 Snapshot 时 Lease 过期引发的异常竞争问题 [#13553](https://github.com/tikv/tikv/issues/13553) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
```

File: releases/release-6.1.3.md [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+
+    - 优化锁的粒度以减少锁争用，提升高并发下心跳的处理能力 [#5586](https://github.com/tikv/pd/issues/5586) @[rleungx](https://github.com/rleungx)
+
+- Tools
+
+    - TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+        - 开启大事务拆分，默认关闭同步任务的安全模式以提升性能 [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
```
Comment (Oreoxmt) [inline releases/release-6.1.3.md:27]:
```suggestion
        - 默认情况下关闭 safeMode 并开启大事务拆分功能，提升同步的稳定性 [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
```
https://github.com/pingcap/docs-cn/pull/12146

File: releases/release-6.1.3.md [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
++ TiFlash
+
+    <!--compute owner: zanmato1984 -->
+
+    - (dup) 修复逻辑运算符在 UInt8 类型下查询结果出错的问题 [#6127](https://github.com/pingcap/tiflash/issues/6127)
+    - (dup) 修复 `CAST(value AS datetime)` 输入数据无法转成 `DATETIME` 时会导致 TiFlash sys CPU 异常高的问题 [#5097](https://github.com/pingcap/tiflash/issues/5097) @[xzhangxian1008](https://github.com/xzhangxian1008)
+
+    <!--storage owner: flowbehappy -->
+
+    - 修复高压力写可能产生太多 delta 层小文件的问题 [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
```
Comment (Oreoxmt) [inline releases/release-6.1.3.md:81]:
```suggestion
    - 修复高压力写入可能产生太多 delta 层小文件的问题 [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: releases/release-6.1.3.md [hunk 23]
```diff
@@ -0,0 +1,117 @@
...
+    - TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+        - 开启大事务拆分，默认关闭同步任务的安全模式以提升性能 [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
+        - 提升 Kafka 相关协议的编码性能 [#7540](https://github.com/pingcap/tiflow/issues/7540), [#7532](https://github.com/pingcap/tiflow/issues/7532), [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
+
+- 其他
+
+    - 为了提升 TiDB 性能，TiDB 的 Go 编译器版本从 go1.x.x 升级到了 go1.19。<!--待提供更多信息>
```
Comment (XuHuaiyu) [inline releases/release-6.1.3.md:32]:
```suggestion
    - 为了提升 TiDB 稳定性，缓解 OOM 问题，TiDB 的 Go 编译器版本从 go1.18 升级到了 go1.19。通过设置环境变量 GOMEMLIMIT，可以 TiDB 的内存使用维持在预定的水位线以下，缓解大部分的 OOM 问题。更多信息详见 <!--待提供更多信息>
```

File: releases/release-6.1.3.md [hunk 24]
```diff
@@ -0,0 +1,75 @@
...
+- Tools
+
+    - TiCDC
+
+        - 默认情况下关闭 safeMode 并开启大事务拆分功能，提升同步的稳定性 [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
+        - 提升 Kafka 相关协议的编码性能 [#7540](https://github.com/pingcap/tiflow/issues/7540) [#7532](https://github.com/pingcap/tiflow/issues/7532) [#7543](https://github.com/pingcap/tiflow/issues/7543) @[sdojjy](https://github.com/sdojjy) @[3AceShowHand](https://github.com/3AceShowHand)
+
+- 其他
+
+    - 为了提升 TiDB 稳定性，缓解 OOM 问题，TiDB 的 Go 编译器版本从 go1.18 升级到了 [go1.19](https://go.dev/blog/go1.19)。通过设置环境变量 `GOMEMLIMIT`，可以将 TiDB 的内存使用维持在预定的水位线以下，缓解大部分 OOM 问题。更多信息，详见[设置环境变量 `GOMEMLIMIT` 缓解 OOM 问题](configure-memory-usage.md#设置环境变量-gomemlimit-缓解-oom-问题)。
```
Comment (Oreoxmt) [inline releases/release-6.1.3.md:28]:
```suggestion
    - 为了提升 TiDB 稳定性，缓解 OOM 问题，TiDB 的 Go 编译器版本从 go1.18 升级到了 [go1.19](https://go.dev/blog/go1.19)。通过设置环境变量 [`GOMEMLIMIT`](https://pkg.go.dev/runtime#hdr-Environment_Variables)，可以将 TiDB 的内存使用维持在预定的水位线以下，缓解大部分 OOM 问题。更多信息，详见[设置环境变量 `GOMEMLIMIT` 缓解 OOM 问题](configure-memory-usage.md#设置环境变量-gomemlimit-缓解-oom-问题)。
```

**Review by bb7133 — APPROVED** [review]
LGTM

File: releases/release-6.1.3.md [hunk 25]
```diff
@@ -0,0 +1,75 @@
...
+
+    - 修复 Stream 超时问题，提高 Leader 切换的速度 [#5207](https://github.com/tikv/pd/issues/5207) @[CabinfeverB](https://github.com/CabinfeverB)
+
++ TiKV
+
+    - 修复获取 Snapshot 时 Lease 过期引发的异常竞争问题 [#13553](https://github.com/tikv/tikv/issues/13553) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
+
++ TiFlash
+
+    - 修复逻辑运算符在 UInt8 类型下查询结果出错的问题 [#6127](https://github.com/pingcap/tiflash/issues/6127)
```
Comment (zanmato1984) [inline releases/release-6.1.3.md:55]:
```suggestion
    - 修复逻辑运算符在 `UInt8` 类型下查询结果出错的问题 [#6127](https://github.com/pingcap/tiflash/issues/6127) @[xzhangxian1008](https://github.com/xzhangxian1008)
```

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Review by nongfushanquan — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**